modules/gapi/src/backends/common/gmetabackend.cpp

```diff
@@ -0,0 +1,105 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2019-2020 Intel Corporation
+
+#include "precomp.hpp"
+
+#include <opencv2/gapi/gcommon.hpp>        // compile args
+#include <opencv2/gapi/util/any.hpp>       // any
+#include <opencv2/gapi/streaming/meta.hpp> // GMeta
+
+#include "compiler/gobjref.hpp"            // RcDesc
+#include "compiler/gmodel.hpp"             // GModel, Op
+#include "backends/common/gbackend.hpp"
+#include "api/gbackend_priv.hpp" // FIXME: Make it part of Backend SDK!
+
+#include "backends/common/gmetabackend.hpp"
+
+namespace {
+
+class GraphMetaExecutable final: public cv::gimpl::GIslandExecutable {
+    std::string m_meta_tag;
+
+public:
+    GraphMetaExecutable(const ade::Graph& g,
+                        const std::vector<ade::NodeHandle>& nodes);
+    bool canReshape() const override;
+    void reshape(ade::Graph&, const cv::GCompileArgs&) override;
+
+    void run(std::vector<InObj> &&input_objs,
+             std::vector<OutObj> &&output_objs) override;
+};
+
+bool GraphMetaExecutable::canReshape() const {
+    return true;
+}
+void GraphMetaExecutable::reshape(ade::Graph&, const cv::GCompileArgs&) {
+    // do nothing here
```

Comment:
Is this an expected behavior?

Reply:
Oh, I see. It's supposed to be a single-node graph

Reply:
Yes. There's nothing to reshape internally so nothing to do here, but we have to indicate we do support reshape in this case to keep chance of the full graph to be reshapable.